Neos.Flow/Configuration/Settings.Object.yaml

```diff
@@ -23,3 +23,8 @@ Neos:
       # by specifying corresponding regular expressions that don't match classes to exclude.
       #
       includeClasses: []
+
+      # 0 means no optimizations get enabled, the higher the number the more optimizations but the more it _could_ be breaking in special use cases.
+      # 1 excludes object serialization helpers for classes that only have simple type properties and have no injections, should be very safe
```

Comment:
Thanks first for investigating the proxy building ^^

Hmmm this could be real bad for the flow eco system, as when i require a package which uses dtos with `new self` (when it was developed on 1) but the project i require it in is on level 0 it woulndt work.

Reply:
That is true I guess. We could just decide on optimizations we want and skip the setting, but I think it might be very helpful to introduce more aggressive optimizations that won't work on any code base necessarily.

Reply:
I'm having a hard time understanding this - could the documentation be updated to tell about this optimized proxy? 
And is there a real usecase of different levels? @mhsdesign points out a case that could lead to different experience (works on my machine-syndrome)

Reply:
yeah, I see this will be a difficult sell 😂 I will remove the setting and just put in the respective optimizations I find sensible 😆

Reply:
I love awesome optimization for everybody/all cases 🚀 <3